neighsyncd/restore_neighbors.py

```diff
@@ -239,6 +240,10 @@ def main():
     warmstart.initialize("neighsyncd", "swss")
     warmstart.checkWarmStart("neighsyncd", "swss", False)
 
+    timeout = warmstart.getWarmStartTimer("bgp", "bgp")
+    if timeout <= 0:
+        timeout = TIME_OUT
+
```

Comment:
the neighbor timeout should not be driven by bgp timeout as if we force neighbor restore to timeout, then we could end up with unresolved entries and once we do reconciliation, we will have delete the neighbor in the asic and disrupt the data plane.

Reply:
Done.